lib/hanami/assets/helpers.rb

```diff
@@ -125,9 +131,11 @@ def self.included(base)
       #   <%= javascript 'application' %>
       #
       #   # <script src="https://assets.bookshelf.org/assets/application-28a6b886de2372ee3922fcaf3f78f2d8.js" type="text/javascript"></script>
-      def javascript(*sources)
+      def javascript(*sources, **options)
```

Comment:
Sorry that I did not make myself clear, what I meant is that instead of doing generic wilcard `**options*`, what do you think about explicitly one hash option `async: <what ever default value it is>`?